docs/core/tutorials/using-on-mac-vs-full-solution.md

```diff
@@ -19,8 +19,8 @@ This tutorial shows you how to create an application that accepts a search word
 
 > [!NOTE]
 > Visual Studio for Mac is preview software. As with all preview versions of Microsoft products, your feedback is highly valued. There are a two ways you can provide feedback to the development team on Visual Studio for Mac:
-> * In Visual Studio for Mac, select **Help > Report a Problem** from the menu or **Report a Problem** from the Welcome screen, which opens a window for filing a bug report.
-> * To make a suggestion, select **Help > Provide a Suggestion** from the menu or **Provide a Suggestion** from the Welcome screen, which takes you to the [Visual Studio for Mac UserVoice webpage](https://visualstudio.uservoice.com/forums/563332-visual-studio-for-mac).
+> * In Visual Studio for Mac, select **Help** > **Report a Problem** from the menu or **Report a Problem** from the Welcome screen, which opens a window for filing a bug report.
+> * To make a suggestion, select **Help** > **Provide a Suggestion** from the menu or **Provide a Suggestion** from the Welcome screen, which takes you to the [Visual Studio for Mac UserVoice webpage](https://visualstudio.uservoice.com/forums/563332-visual-studio-for-mac).
 
 ## Prerequisites
```

Comment:
I think we need to explicitly call out versions and software used here. Perhaps we should say something like
- OpenSSL (if running .NET Core 1.0 or 1.1 versions) -> I believe it's not required for 2.0?
- .NET Core runtime x.y.z and later versions. -> you don't seem to use the sdk here or if the sdk is needed then call it out sdk and version
- Visual Studio 2017 for Mac